docs/introduction/renku.rst

```diff
@@ -0,0 +1,134 @@
+.. _renku:
+
+Renku CLI
+=========
+
+The core of the Renku Project is the ``renku`` command-line interface (CLI),
+which offers tools for easily capturing your data science process as you work.
+With these tools, you can describe and annotate data and workflows, providing
+information that is used to build the lineage of your results, simplifying
+iterative development and making your work reproducible. The CLI can be used
+within Renkulab or locally, on your own machine.
+
+The importance of version control for working with code is widely recognized.
+`renku` aims to be "git for research", by extending version control to encompass
```

Comment:
we discussed this a bit with Emma and we have de-emphasized data science in a few places - basically, the researchers, data owners etc wonder if it doesn't fit them because they don't see themselves as data scientists

Reply:
though I agree it's not obvious what to do  in this case. Personally i'm not sure I really like the "git for research" slogan anyway.

Reply:
renku aims to be "git for code & data workflows"

Reply:
or "data analysis" was the phrase we were throwing around at some point as well